### PR TITLE
Result template issue fix for azurermwebapp and azurefilecopy

### PR DESCRIPTION
**Pull request by Ajay-MS**

### CODE REVIEW

File: Tasks/AzureFileCopy/task.json [hunk 1]
```diff
@@ -13,12 +13,12 @@
...
     "Major": 1,
     "Minor": 0,
-    "Patch": 58
+    "Patch": 59
   },
   "demands": [
     "azureps"
   ],
-    "minimumAgentVersion": "1.98.0",
+    "minimumAgentVersion": "1.102.0",
```
Comment (omeshp) [inline Tasks/AzureFileCopy/task.json]: why is this change required?

**Comment by anaggar** [comment]
please run L0 tests before checkin
